### PR TITLE
Pridejau SavePuzzle funkcija kuri issaugo puzzle objekta sql db

### PR DESCRIPTION
**Pull request by Justuxs**
Prasau pakomentuokite

### CODE REVIEW

File: CityPuzzle/Classes/Sql.cs [hunk 1]
```diff
@@ -63,7 +63,21 @@ public static String ReadUsers()
             }
         }
 
-
+        public static void SavePuzzle(Puzzle puzzle)
+        {
+            using (SqlConnection conn = new SqlConnection(ConnStr))
+            {
+                conn.Open();
+                var command = new SqlCommand("INSERT INTO Puzzles (Name,About,Quest,Latitude,Longitude) VALUES (@Name,@About,@Quest,@Latitude,@Longitude)", conn);
```
Comment (AllBill127) [inline CityPuzzle/Classes/Sql.cs:71]: Could use Entity Framework to save whole object in one command

File: CityPuzzle/Classes/Sql.cs [hunk 2]
```diff
@@ -63,7 +63,21 @@ public static String ReadUsers()
...
+                command.Parameters.AddWithValue("@Name", puzzle.Name);
+                command.Parameters.AddWithValue("@About", puzzle.About);
+                command.Parameters.AddWithValue("@Quest", puzzle.Quest);
+                command.Parameters.AddWithValue("@Latitude", puzzle.Latitude);
+                command.Parameters.AddWithValue("@Longitude", puzzle.Longitude);
```
Comment (AllBill127) [inline CityPuzzle/Classes/Sql.cs:76]: Missing Puzzle.ImgAdress

File: CityPuzzle/Classes/Sql.cs (hunk 1 above)
Comment (Justuxs) [reply CityPuzzle/Classes/Sql.cs:71]: I'll fix it later. Now I want to end with a simple idea of sql input.

File: CityPuzzle/Classes/Sql.cs (hunk 2 above)
Comment (Justuxs) [reply CityPuzzle/Classes/Sql.cs:76]: Ok i will add and will add field to Azure db

**Pull request merged**